modules/apps/adaptive-media/adaptive-media-image-api/src/main/java/com/liferay/adaptive/media/image/scaler/AMImageConvertedImage.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.adaptive.media.image.scaler;
+
+/**
+ * @author Minhchau Dang
+ */
+public interface AMImageConvertedImage {
+
+	public String getMimeType();
+
+}
```

Comment:
@holatuwol Can we get rid of this interface? Why not add the getMimeType method to the AMImageScaledImage one?

Reply:
Hi @adolfopa ,

We considered adding this method, but found limitations especially for backports.

> **Solution notes**
> 
> The solution on https://github.com/holatuwol/liferay-portal/pull/258 modified the AMImageScaledImage interface to add the getMimeType method, which (due to being a ConsumerType interface) resulted in a major version change when running baseline, which we would not be able to backport to earlier branches.